change/@azure-msal-common-2020-10-29-03-03-01-local-account-id.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Fixing a bug and adding `localAcccountId` in AccountInfo interface (#2516)",
+  "packageName": "@azure/msal-common",
+  "email": "sameera.gajjarapu@microsoft.com",
+  "dependentChangeType": "patch",
```

Comment:
```suggestion
  "dependentChangeType": "none",
```

Reply:
I don't understand why this doesn't reflect. I can swear made sure I added "none" this time. :-(